src/main/kotlin/com/safehill/kclient/models/SHRemoteUser.kt

```diff
@@ -36,6 +44,39 @@ data class SHRemoteUser(
         }
     }
 
+    class MapInResultDeserializer : ResponseDeserializable<Map<String, SHRemoteUser>> {
```

Comment:
For consistency with the method below: `MapDeserializer`and `ListDeserializer`

Reply:
Its not really a map as a whole. The map object is contained within "result" key of the json. Do we really want to name in MapDeserializer?

Reply:
Up to you but MapInResult is cryptic. Please find a better name